.github/workflows/dependency-review.yml

```diff
@@ -9,4 +9,6 @@ jobs:
       - name: 'Checkout Repository'
         uses: actions/checkout@v3
       - name: Dependency Review
-        uses: actions/dependency-review-action@v2
+        uses: actions/dependency-review-action@v3
+        with:
+         allow-ghsas: GHSA-mjmj-j48q-9wg2
```

Comment:
Why are we skipping `GHSA-mjmj-j48q-9wg2`?  this relates to snake yaml https://github.com/advisories/GHSA-mjmj-j48q-9wg2

Reply:
@n5nk 
for dependency review action, the option `allow-ghsas` will skip the CVE id mentioned and thus will not block pushing the PR.
 This [CVE](https://github.com/advisories/GHSA-mjmj-j48q-9wg2) (snakeyaml)  is skipped because this do not have any patched version [available](https://github.com/atlanhq/atlas-metastore/security/dependabot/142) and it is blocking other snakeyaml CVE (6) to land which has patched version available and can get fixed.

We will revert `allow-ghsas`  option back to original once the fix lands.